sda/cmd/auth/main.go

```diff
@@ -263,6 +264,10 @@ func (auth AuthHandler) elixirLogin(ctx iris.Context) *OIDCData {
 
 		return nil
 	}
+	err = auth.Config.DB.UpdateUserInfo(idStruct.User, idStruct.Name, idStruct.Email, idStruct.EdupersonEntitlement)
+	if err != nil {
+		log.Warnf("Could not log user info for %s (%s, %s)", idStruct.User, idStruct.Name, idStruct.Email)
```

Comment:
```suggestion
		log.Warnln("Could not log user info")
```

Do not expose the user identity in the logs